params/params_EShalom.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4.568
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Eve's Parakeet "
+    
+# e.g. "red"    
+color <- " darkorchid4 "
```

Comment:
Removing spaces in case of problems

```suggestion
color <- "darkorchid4"
```